src/sql/workbench/contrib/editData/browser/gridParentComponent.ts

```diff
@@ -72,31 +72,32 @@ export abstract class GridParentComponent {
 	protected _messageActive = true;
 	protected activeGrid = 0;
 
-	@ViewChildren('slickgrid') slickgrids: QueryList<SlickGrid>;
+
+	protected nativeElement: HTMLElement;
+	protected _tables: Table<any>[] = [];
```

Comment:
does this really need to be an array?  I see us referencing directly to item `0` all over the place, but never actually going through all the items in the array.

Reply:
For now, it will be an array, at the end it won't.